src/sources/soundsourcem4a.cpp

```diff
@@ -60,6 +60,16 @@ constexpr SINT kNumberOfPrefetchFrames = 2112;
 // codecs AAC-LD and AAC-ELD."
 constexpr int kDefaultFrameSize = 1024;
 
+// According to various references DecoderConfigDescriptor.bufferSizeDB
+// is a 24-bit unsigned integer value.
+// MP4 atom:
+//   trak.mdia.minf.stbl.stsd.*.esds.decConfigDescr.bufferSizeDB
+// References:
+//   https://github.com/sannies/mp4parser/blob/master/isoparser/src/main/java/org/mp4parser/boxes/iso14496/part1/objectdescriptors/DecoderConfigDescriptor.java
+//   http://mutagen-specs.readthedocs.io/en/latest/mp4/
+//   http://perso.telecom-paristech.fr/~dufourd/mpeg-4/tools.html
+constexpr u_int32_t kMaxSampleBlockInputSizeLimit = (u_int32_t(1) << 24) - 1;
```

Comment:
What is the difference between `u_int32_t` and `uint32_t` (used below in line 525)? Let's use a single naming. (maybe `quint2`?)